Sources/Commands/SwiftBuildTool.swift

```diff
@@ -100,6 +119,30 @@ public class SwiftBuildTool: SwiftTool<BuildToolOptions> {
             print("warning: minimum recommended clang is version 3.6, otherwise you may encounter linker errors.")
         }
     }
+
+    private static let replLibName = "__SwiftReplLibrary"
+
+    private func addReplLibrary(to manifest: Manifest) -> Manifest {
```

Comment:
Instead of creating an entirely new manifest, we can just add a new product in the root package itself. This needs to happen in the `PackageBuilder` class. The loadPackageGraph method in Workspace can take a bool "createREPLProduct" which can be passed down to PackageGraphLoader and then finally to PackageBuilder. We currently have similar-ish thing which is controlled by the `createMultipleTestProducts` boolean in the loadPackageGraph method. Let me know if this makes sense.

Reply:
I'll take a look. 👍